registration-processor/pre-processor/registration-processor-packet-validator-stage/src/test/java/io/mosip/registration/processor/stages/validator/PacketValidatorImplTest.java

```diff
@@ -283,6 +310,12 @@ public void testBiometricsXSDValidatonFailure() throws Exception {
 		doThrow(new CbeffException("CbeffException occurred")).when(biometricsXSDValidator).validateXSD(any());
 		assertFalse(PacketValidator.validate("123456789", "NEW", packetValidationDto));
 	}
+	
+	@Test(expected = BaseCheckedException.class)
+	public void testBiometricsSignatureValidatonFailure() throws Exception {
+		doThrow(new BaseCheckedException("JWT signature Validation Failed")).when(biometricsSignatureValidator).validateSignature(any());
+		assertFalse(PacketValidator.validate("123456789", "NEW", packetValidationDto));

```

Comment:
Why do we have both exception expected and assert to false ?

Reply:
removed exception expected and only checking assert to false.